frameworks/hdfs/universe/resource.json

```diff
@@ -4,6 +4,7 @@
       "jre-tar-gz": "{{jre-jce-unlimited-url}}",
       "libmesos-bundle-tar-gz": "{{libmesos-bundle-url}}",
       "hdfs-tar-gz": "https://downloads.mesosphere.com/hdfs/assets/hadoop-2.6.0-cdh5.11.0.tar.gz",
+      "hdfs-jre-tar-gz": "{{jre-jce-unlimited-url}}",
```

Comment:
This appears to be the same value as `jre-tar-gz` above. Are these expected to vary in practice?